git/src/pages/Events.tsx

```
@@ -0,0 +1,3 @@
+tools:
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)